### ISSUE TITLE
Migrating Run2.2i data at NERSC from projecta to CFS

### ISSUE THREAD

**Issue by heather999** [LSSTDESC/ImageProcessingPipelines#143]
@airnandez now that NERSC has set up the Community File System. We will likely be migrating the data that is currently stored on `/global/projecta/projectdirs/lsst` to the new area `/global/cfs/cdirs/lsst`.  In the context of Run2.2i, this would include the `y1-wfd` and `y2-wfd` data already transferred to CC and require us to update the tracking DB.
Do you anticipate this would cause problems on your end?  
Ultimately this will affect all the data already transferred over for all of DC2, but I suspect we consider Runs 1.1p, 1.2i, 1.2p, 2.1i, 2.1.1i in some sense "done", so those may not matter so much.

**Comment by airnandez** [comment]
No, that relocation of the data at NERSC should not be a problem. We just need that the tracking database we use for determining what files should be transferred contains the path of each visit new top directory.
